lib/acme/client/jwk/base.rb

```diff
@@ -0,0 +1,77 @@
+class Acme::Client::JWK::Base
```

Comment:
> That matches the subclasses defined in this repo, but isn't very general.

Ah fair enough. I assumed based on the ones implemented that callers could expect a similar initialization from subclasses of `Acme::Client::JWK::Base` but if it isn't generalisable then you're probably right to leave it out.